hack/deploy-image.sh

```diff
@@ -60,6 +60,7 @@ rc=0
 for ii in $( seq 1 5 ) ; do
     if push_image ${IMAGE_TAG} ${tag} ; then
         rc=0
+        oc -n openshift get imagestreams | grep cluster-logging-operator
```

Comment:
Does this mean that `push_image` can somehow succeed, but the image isn't actually pushed?

Reply:
No.  I added this as a convenience when manually pushing your images so you could see the full pull spec as an output in the execution of the target.  Suggestions how to make it more obvious?  Maybe an echo or even capturing the output and spliting/combining it into an actual pull spec?

Reply:
No, I think this is fine.